.babelrc

```diff
@@ -1,6 +1,6 @@
 {
   "presets": [
-    ["@babel/preset-env", { "modules": false }],
+    "@babel/preset-env",
```

Comment:
Would you mind explaining this change? - It's not mentioned in the commit message.

Reply:
Well, it makes it work!

Reply:
For posterity: https://babeljs.io/docs/en/babel-preset-env#modules

Reply:
I wonder what you would have done that would break the default of `auto`.

Have you tested recently to see whether it is still needed?

Reply:
I have now, and there's still an error:
```
npm test

> lev-react-components@0.8.0 test /home/robin/dev/HomeOffice/lev-react-components
> jest

 FAIL  test/lev-report/charts/HourlyUsage.test.js
  ● Test suite failed to run

    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    /home/robin/dev/HomeOffice/lev-react-components/test/lev-report/charts/HourlyUsage.test.js:1
    import React from 'react';
    ^^^^^^

    SyntaxError: Cannot use import statement outside a module

      at Runtime._execModule (node_modules/jest-runtime/build/index.js:1010:58)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        2.333s, estimated 4s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
```

Reply:
Okay. Looks like maybe jest needed some babel config but we can worry about this later as long as the change doesn't break things in the browser. (Or are we strictly SSR at the moment?)